bundles/org.openhab.binding.pushover/src/main/java/org/openhab/binding/pushover/internal/handler/PushoverAccountHandler.java

```diff
@@ -66,7 +69,15 @@ public void handleCommand(ChannelUID channelUID, Command command) {
     @Override
     public void initialize() {
         config = getConfigAs(PushoverAccountConfiguration.class);
-
+        if (!httpClient.isStarted()) {
```

Comment:
See comment above. This check would not be needed if life-cycle is managed by the handler factory.